src/frontend/csrs-portal/src/app/components/landing/landing.component.scss

```diff
@@ -0,0 +1,589 @@
+@import "palette";
+@import "bootstrap/bootstrap-imports";
+
+h1 {
+  font-size: 3em;
+  line-height: 1.1em;
+  margin-bottom: 50px;
+  font-weight: bold;
+  text-shadow: 2px 2px 4px #000;
+}
+
+h3 {
+  font-size: 1.875em;
+  line-height: 1.4em;
+  font-weight: bold;
+  margin-bottom: 10px;
+  color: theme-palette(blue);
+}
+
+h4 {
+  font-size: 1.5em;
+  font-weight: bold;
+  margin-bottom: 20px;
+  color: theme-palette(blue);
+}
+
+.lead {
+  font-size: 1.5em;
+  margin-bottom: 10px;
+  line-height: 1.5em;
+  text-shadow: 2px 2px 4px #000;
+}
+
+@media (max-width: 320px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-320x500.jpg") no-repeat !important;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 321px) and (max-width: 375px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-375x420.jpg") no-repeat !important;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 376px) and (max-width: 414px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-414x420.jpg") no-repeat !important;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 415px) and (max-width: 600px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-600x420.jpg") no-repeat !important;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 601px) and (max-width: 767px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-768x420.jpg") no-repeat !important;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 992px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background-992x420.jpg") no-repeat;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+@media (min-width: 993px) {
+  div.landing-summary {
+    background: url("/assets/landing-page-background.png") no-repeat;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+  }
+}
+
+.summary-header-links {
+  list-style: none;
+  margin-bottom: 0;
+  text-align: center;
+  padding: 0;
+
+  li {
+    line-height: 2em;
+  }
+
+  a {
+    font-size: 1.125em;
+    color: #fff !important;
+    font-weight: 400;
+  }
+}
+
+.divider {
+  color: #fff;
```

Comment:
can you make use of theme palette as above like " color: theme-palette(blue); "
where ever necessary.

Reply:
Replacement creates error.